lib/Wikia/tests/Service/Helios/HeliosClientTest.php

```diff
@@ -7,25 +7,22 @@
 
 class HeliosClientTest extends \WikiaBaseTest {
 
-	public function setUp()
-	{
+	public function setUp() {
 
 		parent::setUp();
 	}
 
-	public function testCannotMakeRequests()
-	{
-		$this->setExpectedException('Wikia\Util\AssertionException');
+	public function testCannotMakeRequests() {
+		$this->setExpectedException( 'Wikia\Util\AssertionException' );
 
 		$this->mockStaticMethod( '\MWHttpRequest', 'canMakeRequests', false );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "canMakeRequests" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\MWHttpRequest" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)